### PR TITLE
BC-1417-taskoveriew-update-test-cases

### PR DESCRIPTION
**Pull request by UzaeirKhan**
# Description
Update the test cases for copy and delete options in task overview menu.

## Links to Tickets or other pull requests
https://ticketsystem.hpi-schul-cloud.org/browse/BC-1417

## Approval for review
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definition of Done
More and detailed information on the *definition of done* can be found [on Confluence](https://docs.hpi-schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

File: page-objects/pages/TASKListPage.js [hunk 1]
```diff
@@ -106,12 +106,6 @@ async function clickActionFromMenuOnTask(button) {
 	await elementHelpers.clickAndWait(actionButton);
 }
 
-async function getTaskDescription() {
```
Comment (d3n153) [inline page-objects/pages/TASKListPage.js:109]: Can you please check in your team if they have unit tests to check if the description is shown. If yes, we don't need to check it as well, if not then we should check if it's there.

File: page-objects/pages/TASKListPage.js [hunk 2]
```diff
@@ -31,8 +31,8 @@ const taskActionMenuButton = {
 	archive: "//*[text()[contains(.,'Abschließen')]]",
 	unarchive: "//*[text()[contains(.,'Wiederherstellen')]]",
 	edit: "//*[text()[contains(.,'Bearbeiten')]]",
-	copy: '.fa-copy',
-	delete: '.btn-delete',
+	copy: "//*[text()[contains(.,'Kopieren')]]",
+	delete: "//*[text()[contains(.,'Löschen')]]",
```
Comment (d3n153) [inline page-objects/pages/TASKListPage.js:35]: Is the deleteTask.feature already adjusted, so that is uses the new tasks page in nuxt?

Comment (UzaeirKhan) [reply page-objects/pages/TASKListPage.js:38]: I guess we discussed it already.

File: page-objects/pages/TASKListPage.js (hunk 1 above)
Comment (UzaeirKhan) [reply page-objects/pages/TASKListPage.js:111]: I asked in my team and yes they check it via unit tests.

https://github.com/hpi-schul-cloud/schulcloud-server/blob/55b82f8b553c3962c3733acd498795cc8e1d575a/apps/server/src/shared/domain/entity/task.entity.spec.ts#L577-L591

Comment (d3n153) [reply page-objects/pages/TASKListPage.js:111]: Perfect, thx :)

**Comment by CeEv** [comment]
@d3n153 can you please review it, that we can finished it in sprint time.

File: package.json [hunk 3]
```diff
@@ -48,7 +48,7 @@
 		"browserstack-local": "^1.4.8",
 		"chai": "^4.3.4",
 		"clean": "^4.0.2",
-		"commander": "^8.2.0",
+		"commander": "^8.3.0",
```
Comment (d3n153) [inline package.json]: Have you pushed the changes to package files on purpose?

**Review by d3n153 — APPROVED** [review]

File: package.json (hunk 3 above)
Comment (d3n153) [reply package.json]: This change will be done in a separate PR.

Comment (UzaeirKhan) [reply package.json]: At my local it needs to update otherwise it was not getting further.

Comment (d3n153) [reply package.json]: The feel free to do so, but please do it in a separate PR. In other repos such things are also done separately from another.

**Review by d3n153 — APPROVED** [review]

**Pull request merged**